### ISSUE TITLE
Latest Update (v3.6.4) type error message and AVR no longer responsive

### ISSUE THREAD

**Issue by kwhittenberger** [grzegorz914/homebridge-denon-tv#69]
Just updated to the latest push and my Marantz AVR is no longer responsive.  Here is the tracing from the log:

2/18/2021, 8:58:30 AM [DenonTv] Device: 192.168.1.31 Marantz AVR Main Zone, state: Online.
2/18/2021, 8:58:30 AM [DenonTv] -------- Marantz AVR --------
2/18/2021, 8:58:30 AM [DenonTv] Manufacturer: Marantz
2/18/2021, 8:58:30 AM [DenonTv] Model: SR6013
2/18/2021, 8:58:30 AM [DenonTv] Zones: 2
2/18/2021, 8:58:30 AM [DenonTv] Api version: 0301
2/18/2021, 8:58:30 AM [DenonTv] Serialnr: 0005CDF96D98
2/18/2021, 8:58:30 AM [DenonTv] Firmware: 00
2/18/2021, 8:58:30 AM [DenonTv] ----------------------------------
2/18/2021, 8:58:30 AM [DenonTv] Device: 192.168.1.31 Marantz AVR Main Zone, update Device state error: TypeError: this.televisionService.getCharacteristic(...).on(...).onSet is not a function
    at denonTvDevice.prepareAccessory (/hoobs/node_modules/homebridge-denon-tv/index.js:337:5)
    at denonTvDevice.updateDeviceState (/hoobs/node_modules/homebridge-denon-tv/index.js:288:8)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

**Comment by grzegorz914** [comment]
Need homebridge 1.3 or abowe

**Comment by grzegorz914** [comment]
You can install previous Version using Config UI X

**Comment by grzegorz914** [comment]
Or update HB to latest beta 1.3.x

**Comment by kwhittenberger** [comment]
I think since I can't upgrade HB due to using HOOBs I am probably going to look at migrating to Homebridge directly and moving away from HOOBs.  Any chance this is something that can be resolved without the HB version requirement?  Thanks.

**Comment by grzegorz914** [comment]
No, just move to HB and Config UI X
https://github.com/homebridge/homebridge-raspbian-image/wiki/How-to-migrate-from-HOOBS-to-Homebridge

**Comment by kwhittenberger** [comment]
Ok, I have migrated to running Homebridge directly on Docker, but am sticking with the latest release.  I have successfully switched to an older version of the plug-in that is compatible and will stay there until HB itself is updated.  I appreciate the help.

A suggestion though for pushing updated releases that require a beta version: anyone is going to see this as an available update and may not realize the incompatibility until it is too late, especially if they are on HOOBs.  Not sure how to make these available in this ecosystem, but it will be a source of conflict.


Thanks again
